module-cellular/modem/ATCommon.cpp

```diff
@@ -156,3 +162,108 @@ Result Channel::checkResult(bsp::cellular::CellularResultCode cellularResult)
 
     return result;
 }
+
+std::string Channel::anonymizeIfNecessary(std::string text)
+{
+    std::string substringCPIN = "CPIN";
+    std::string substringCLCC = "CLCC";
+    std::string substringATD  = "ATD";
+    std::string substringCPBR = "CPBR";
+    std::string substringCSPN = "CSPN";
+
+    if (text.find(substringCPIN) != std::string::npos) {
+        text = anonymizeInQuotationMarks(text);
```

Comment:
You are using else if
then you can just return when calling the function
if(){
return func():
}
else if()
return func1():
}